src/interfaces.ts

```diff
@@ -98,9 +98,9 @@ export interface IDocsOptions {
    */
   prefix?: string;
   /**
-   * Whether to omit the URL prefix from the generated documentation.
+   * Prefix to omit from the generated documentation.
    */
-  shouldOmitPrefixInDocs?: boolean;
+  prefixToOmit?: string;
```

Comment:
Should we continue to support the old one too? Would just be a few lines of extra code that might immediately be unused, so unsure its worth adding.